src/style/style.js

```diff
@@ -0,0 +1,64 @@
+import styled from "styled-components";
```

Comment:
this is not a styles  , they are styled components. and  should be stored in common  components folder. in a separate files